third_party/BUILD

```diff
@@ -26,8 +26,6 @@ filegroup(
         "//third_party/java/jdk/langtools:srcs",
         "//third_party/java/proguard:srcs",
         "//third_party/javascript/bootstrap:srcs",
-        "//third_party/checker_framework_dataflow:srcs",
```

Comment:
The targets of `third_party:srcs` are not all embedded into the bazel binary. IIUC this target is to keep track of the sources in the bazel repo, so I would keep them here.

Reply:
Done -- I removed it just so that rdeps gives a cleaner answer.